src/components/Modal/staking-hub/DockerRunCommandModal.tsx

```
@@ -58,58 +64,260 @@ export const DockerRunCommandModal = (props: DockerRunCommandModalProps) => {
+        <a
+          href="https://docs-hoprnet-org-git-experiment-ui-tweak-300-hoprnet.vercel.app/node/node-dappnode"
+          target="_blank"
+          rel="noreferrer"
+          style={{ color: '#007bff', textDecoration: 'underline' }}
+        >
+          Checkout the full instructions in our docs
+        </a>
+      </div>
+    </SDialogContent>
+  )
+};
```

ESLint

[error] Unnecessary semicolon.

(no-extra-semi)